website/docs/d/private_endpoint_regional_mode.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint_regional_mode"
+sidebar_current: "docs-mongodbatlas-datasource-private-endpoint-regional-mode"
+description: |-
+    Describes a Private Endpoint Regional Mode
+---
+
+# private_endpoint_regional_mode
+
+`private_endpoint_regional_mode` describe a Private Endpoint Regional Mode. This represents a Private Endpoint Regional Mode Connection that wants to retrieve settings of an Atlas project.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+## Example Usage
+
+```terraform
+resource "private_endpoint_regional_mode" "test" {
+  project_id    = "<PROJECT-ID>"
+  enabled       = true
+}
+
+data "private_endpoint_regional_mode" "test" {
+	project_id = private_endpoint_regional_mode.test.project_id
+	enabled    = true
```

Comment:
shouldn't it return this value, not need it?

Reply:
Good catch, this was actually indicative of a larger issue in the codebase, more than just the documentation.

Code, tests, and documentation have been fixed.